### PR TITLE
fixed logger and made fat jar (works on robot)

### PR DESCRIPTION
**Pull request by matt-mekha**

### CODE REVIEW

**Review by DGKSK8LIFE — CHANGES_REQUESTED** [review]
```kt
 } catch (e: Exception) {
                System.err.println("[Logger] Could not save log files!")
 }
```
Could you also add the exception to the error print? Ty

**Comment by matt-mekha** [comment]
> ```kotlin
>  } catch (e: Exception) {
>                 System.err.println("[Logger] Could not save log files!")
>  }
> ```
> 
> Could you also add the exception to the error print? Ty

I figured if the Logger wasn't working we wouldn't wanna use the logger.

**Comment by matt-mekha** [comment]
@DGKSK8LIFE actually i changed my mind, see latest change

File: .git-hooks/pre-commit [hunk 1]
```diff
@@ -23,6 +23,6 @@ if [[ $(echo $illegal_files | wc -l) != 1 ]]; then
     exit 1
 fi
 
-# If there are whitespace errors, print the offending file names and fail.
-echo "Removing spaces at ends of lines"
-exec find botplus/src -type f -print0 | xargs -0 perl -pi -e 's/ +$//'
+# # If there are whitespace errors, print the offending file names and fail.
+# echo "Removing spaces at ends of lines"
+# exec find botplus/src -type f -print0 | xargs -0 perl -pi -e 's/ +$//'
```
Comment (AlexDvorak) [inline .git-hooks/pre-commit:28]: just delete these lines, they're pointless now with ktlint

**Review by AlexDvorak — CHANGES_REQUESTED** [review]

**Review by AlexDvorak — APPROVED** [review]

**Pull request merged**